frontend/src/css/md.css

```diff
@@ -221,11 +222,11 @@ button .prose.prose {
   border: none;
 }
 
-.prose ins.break:before,
-.prose del.break:before {
+.prose ins.break::before,
+.prose del.break::before {
   content: "\00a0\b6\00a0";
-  -webkit-border-radius: 3px;
-  -moz-border-radius: 3px;
+  border-radius: 3px;
+  border-radius: 3px;
   border-radius: 3px;
```

Comment:
These changes (here and in other places) make no sense and might break some backwards compatibility.

Reply:
Our postcss (should) handle browser compatibility / automatically add prefixes

Reply:
Nice 👍🏻 Still, I just wanted to point out the inconsistencies in this AI PR. These lines now appear x3 and have nothing to do with the stated purpose of the PR. 
Anyway, big fan of marimo, keep up the good work!

Reply:
Yea I am unsure if it was the AI that did this or our stylelint tool (that the AI used). Agreed it's now duplicated but is harmless (and cleaned up later)